claim/schema.py

```diff
@@ -235,6 +241,15 @@ def resolve_claim_officers(self, info, search=None, **kwargs):
             )
         return qs
 
+    def resolve_fsp_from_claim(self, info, **kwargs):
+        if not info.context.user.has_perms(ClaimConfig.gql_query_claim_officers_perms):
+            raise PermissionDenied(_("unauthorized"))
+        result = Insuree.objects.filter(
+            chf_id=kwargs['insuree_code'],
+            validity_from__lte=kwargs['date_claimed'],
```

Comment:
```suggestion
        result = Insuree.objects.filter(
            chf_id=kwargs['insuree_code'],
            *filter_validity(validity=kwargs['date_claimed']),
```

Reply:
I am doing a different check. This query checks FSP at the time of creation of a claim. So it can be 2 years old and have 5 changes in the meantime. That's why I am fetching insurees past certain date, sort them and take the last one into account.

Reply:
I think you are right, I've changed the code accordingly.